tedana/due.py

```diff
@@ -10,7 +10,7 @@
 installed duecredit.
 Then use in your code as
     from .due import due, Doi, BibTeX
-See  https://github.com/duecredit/duecredit/blob/master/README.md for examples.
+See  https://github.com/duecredit/duecredit/blob/main/README.md for examples.
```

Comment:
Will need to revert to resolve:
```suggestion
See  https://github.com/duecredit/duecredit/blob/master/README.md for examples.
```